src/AppBundle/Repository/AquariumRepository.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Aquarium;
+use AppBundle\Entity\FishInAquarium;
+use AppBundle\Entity\FishType;
+use AppBundle\Entity\FloatRange;
+use AppBundle\Entity\Gadget\WaterAbsorbGadget;
+use AppBundle\Entity\Gadget\WaterExchangeGadget;
+use AppBundle\Entity\Gadget\WaterMeasuringGadget;
+
+class AquariumRepository {
```

Comment:
How about breaking down this repository?
Currently you have FishTypes and Aquarium managed in one repository.

Reply:
I decided not to break down the repository at this point, I added a comment explaining why.